### ISSUE TITLE
[MicroWin] Sound precentage resets itself every time.

### ISSUE THREAD

**Issue by Lymsleya** [ChrisTitusTech/winutil#2747]
## Describe the bug

Let's say Google Chrome had it's sound to 100% and you set it to 30% (or any percentage) in Volume Mixer, if you close and re-open Google Chrome the sound goes back to 100% by itself no matter the application.

## Expected behavior

The sound should not reset itself at all.

## Screenshots

No need to I think

## Additional context

So the problem is, after you make your ISO with MicroWin and install the windows, if you set your volume mixer to any application/program after you close it or restart/shutdown your PC the to the specific application or even all of them the sound goes back to 100% every time and it becomes extremely annoying to deal with no matter the Windows Version (10/11).

**Comment by DELZORR** [comment]
Had this happen twice already.
Something overzealous deletes mentioned regkey
https://www.reddit.com/r/ChrisTitusTech/comments/1e656ex/sound_mixer_not_saving_individual_app_sound/

**Comment by Mihuy1** [comment]
Also having this issue with fresh install of Windows 11 using MicroWin

**Comment by belthesar** [comment]
> Had this happen twice already. Something overzealous deletes mentioned regkey https://www.reddit.com/r/ChrisTitusTech/comments/1e656ex/sound_mixer_not_saving_individual_app_sound/

Can confirm that this happened to me, and also that creating a registry key at 
`Computer\HKEY_CURRENT_USER\Software\Microsoft\Internet Explorer\LowRegistry\Audio\PolicyConfig\PropertyStore`
allowed the OS to persist those setting as appropriate. For clarity, the key at `Computer\HKEY_CURRENT_USER\Software\Microsoft\Internet Explorer\LowRegistry\` does not exist at all on a fresh MicroWin 11 installation. It does look like that https://github.com/ChrisTitusTech/winutil/pull/2856 adding this key will likely resolve this issue. 

I'll update this comment if the problem described in https://github.com/ChrisTitusTech/winutil/issues/2776 which say that this fix resolves some, but not all issues with regards to Per Application Volume.

**Comment by Mystream90** [comment]
> It does look like that #2856 adding this key will likely resolve this issue.

It does, partially. 
However, if I receive a notification from any application, the system sound resets again, which is still quite annoying. The "solution" for that is to disable notifications. I also noticed that Discord's volume keeps resetting as well. It seems to depend on the application.

**Comment by belthesar** [comment]
Based on some chatter on the [Level1Techs forum](https://forum.level1techs.com/t/windows-11-per-application-volume-mixer-settings-reset-on-application-close-heres-how-to-fix/218134) this may have been a premature change based on a registry key path change between 23H2 and 24H2. On 24H2, the proper keys that save this are at: 

`HKEY_CURRENT_USER\Software\Microsoft\Multimedia\Audio\PolicyConfig\PropertyStore`
and 
`HKEY_CURRENT_USER\Software\Microsoft\Multimedia\Audio\DefaultEndpoint`.

**Comment by Mystream90** [comment]
> Based on some chatter on the [Level1Techs forum](https://forum.level1techs.com/t/windows-11-per-application-volume-mixer-settings-reset-on-application-close-heres-how-to-fix/218134) this may have been a premature change based on a registry key path change between 23H2 and 24H2. On 24H2, the proper keys that save this are at:
> 
> `HKEY_CURRENT_USER\Software\Microsoft\Multimedia\Audio\PolicyConfig\PropertyStore` and `HKEY_CURRENT_USER\Software\Microsoft\Multimedia\Audio\DefaultEndpoint`.

Doesn't work for me.
The behavior of the sound mixer is inconsistent on my side. It does save the volume I set, but if I receive a notification from an application besides windows (which I can trigger with SyncTrayzor), the notification sound plays at 100% volume. Checking the mixer immediately after the notification shows the slider at 100%, but after closing and reopening the mixer, the sound volume returns to my saved setting.

**Comment by CodingWonders** [comment]
Fixed with #2853. /close

**Comment by jflores-p** [comment]
Hi, arriving a bit late, but i used the utility with the 25h2 iso and I'm getting the same behavior, with brave and discord notifications in my case (the main sound of discord does get saved, but the muted/deafe/etc sounds do reset after relaunch)
System sounds and Lost Ark audio are being saved so far.

I guess this could be because of the new ISO, but not so sure, not so savvy in this regard.

I have some entries in 'Computer\HKEY_CURRENT_USER\Software\Microsoft\Internet Explorer\LowRegistry\Audio\PolicyConfig\PropertyStore' and see some updates in the corresponding folders for system sounds and Lost Ark.
I tried hard copying the settings of system sounds in the folder corresponding to brave but windows doesn't seem to read it. Not sure if it would work regardless, I just inferred it.

Any tips or recommendations? thanks :D
